src/stats/inc/LikelihoodBase.h

```diff
@@ -45,7 +46,7 @@ class GslMatrix;
  */
 
 template <class V = GslVector, class M = GslMatrix>
-class BaseGaussianLikelihood : public BaseScalarFunction<V, M> {
+class LikelihoodBase : public BaseScalarFunction<V, M> {
```

Comment:
That is correct. My thinking is the user may want to use this interface for some other likelihood function that's not Gaussian. Thus the name would be more consistent, IMHO.